### PR TITLE
Some small cleanups

### PR DESCRIPTION
**Pull request by milliams**
I've made a few small cleanups to the code which I hope are uncontroversial. This is mostly so that everyone can see what a pull request looks like and practice with commenting and doing the code review.

Once everyone is happy, then we can accept it and make sure that is working.

We don't have any automated checking of pull requests yet but that will be coming in the future.

### CODE REVIEW

**Comment by rob-c** [comment]
I was under the impression that we (core devs) were just dropping code into the develop branch or has this changed?

**Comment by alexanderrichards** [comment]
@rob-c for small bugfixes/typos/the like then simply drop them into develop as it is tested regularly anyway however for anything that needs a branch the merge should go via pull request as it's easier to follow and review
@milliams case in point, I didn't recall we had any globals left but it's much easier for everyone to see here.

**Comment by alexanderrichards** [comment]
@milliams Nice to see the Status API working, what is the QuantifiedCode Check though?

File: python/Ganga/GPIDev/Lib/Tasks/ITransform.py [hunk 1]
```diff
@@ -27,7 +27,6 @@ class ITransform(GangaObject):
         'unit_merger': ComponentItem('mergers', defvalue=None, load_default=0, optional=1, doc='Merger to be copied and run on each unit separately.'),
         'copy_output': ComponentItem('datasets', defvalue=None, load_default=0, optional=1, doc='The dataset to copy all units output to, e.g. Grid dataset -> Local Dataset'),
         'unit_copy_output': ComponentItem('datasets', defvalue=None, load_default=0, optional=1, doc='The dataset to copy each individual unit output to, e.g. Grid dataset -> Local Dataset'),
-        'unit_merger': ComponentItem('mergers', defvalue=None, load_default=0, optional=1, doc='Merger to be run copied and run on each unit separately.'),
```
Comment (alexanderrichards) [inline python/Ganga/GPIDev/Lib/Tasks/ITransform.py]: In the spirit of this pull request (**PR**), namely to illustrate the powerful code reviewing tools available through github, I though I'd show people what an in-line code comment looks like. What follows is a stupid I'm sure question...sorry Matt.

@milliams the title of the commit that this change comes from bc7d4d6cc9b281cdbccd6492083900ff11c2938d is `Remove duplicate dictionary keys` however I don't see the duplicate in this case. Do we definitely want to remove the unit_merger attribute?

File: python/Ganga/GPIDev/Lib/Tasks/ITransform.py [hunk 2]
```diff
@@ -27,7 +27,6 @@ class ITransform(GangaObject):
         'unit_merger': ComponentItem('mergers', defvalue=None, load_default=0, optional=1, doc='Merger to be copied and run on each unit separately.'),
```
Comment (egede) [inline python/Ganga/GPIDev/Lib/Tasks/ITransform.py]: @alexanderrichards Line 27 contains the duplicate.

Comment (alexanderrichards) [inline python/Ganga/GPIDev/Lib/Tasks/ITransform.py]: *Bingo* thanks Ulrik :smile:

**Comment by milliams** [comment]
@alexanderrichards QuantifiedCode is a PyLint-type service which does static code checks. I'm not suggesting that we worry too much about it but it's useful for seeing whether a pull request introduces any _new_ problems.

I will close and accept this pull request now that people have seen how it works.

**Pull request merged**